Soccar [Game-Server]/src/nl/soccar/gameserver/controller/rmi/GameServerRmiController.java

```diff
@@ -26,24 +29,54 @@
     private IMainServerForGameServer mainServerForGameServer;
     private GameServerForMainServer gameServerForMainServer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)